workflow/tests/base_results/results.csv

```diff
@@ -250,7 +251,7 @@ base-hvac-stove-wood-pellets-only.xml,30.46,30.46,0.0,0.0,0.0,0.0,15.71,0.0,0.0,
 base-hvac-undersized.xml,32.25,32.25,8.81,0.0,0.0,0.0,0.0,0.0,0.0,0.23,1.42,0.18,9.15,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,8.81,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,18.4,11.21,9.22,0.61,0.0,0.0,10.09,7.97,6550.0,6440.0,22.55,16.82,0.0,3.71,3.74,0.54,4.11,1.86,-0.16,0.0,0.0,3.14,-0.1,5.43,0.0,0.49,0.0,7.02,-11.49,0.0,0.04,-0.38,-0.01,3.79,-0.29,9.39,0.0,0.0,-5.88,-0.04,-1.12,-6.65,-0.11,0.0,2.76,9.8,1354.0,998.0,11562.0,2653.0
 base-hvac-wall-furnace-elec-only.xml,42.03,42.03,0.0,0.0,0.0,0.0,0.0,0.0,11.62,0.0,0.0,0.0,9.13,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,11.61,0.0,9.22,0.6,0.0,0.0,0.0,0.0,8666.0,0.0,18.4,0.0,0.0,3.95,3.72,0.54,4.13,1.85,-0.13,0.0,0.0,3.11,-0.11,5.41,0.0,0.49,0.0,0.0,-11.44,0.0,-0.24,-0.76,-0.07,3.56,-0.49,8.27,0.0,0.0,-6.59,-0.06,-1.43,-11.78,-0.15,0.0,0.0,9.84,1354.0,998.0,11561.0,2653.0
-base-lighting-ceiling-fans.xml,35.54,35.54,16.05,0.0,0.0,0.0,0.0,0.0,0.0,0.42,3.48,0.69,9.14,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.52,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,16.05,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,15.18,11.32,9.22,0.61,0.0,0.0,0.0,0.01,6575.0,6783.0,23.87,21.64,0.0,3.81,3.73,0.54,4.11,1.86,-0.11,0.0,0.0,3.16,-0.11,5.42,0.0,0.49,0.0,3.65,-11.46,0.0,-0.02,-0.44,-0.02,3.73,-0.32,9.3,0.0,0.0,-5.99,-0.06,-1.16,-6.91,-0.11,0.0,2.99,10.36,1354.0,998.0,11562.0,2653.0
+base-lighting-ceiling-fans.xml,35.77,35.77,16.06,0.0,0.0,0.0,0.0,0.0,0.0,0.42,3.67,0.74,9.15,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.52,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,16.06,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,15.19,12.07,9.22,0.61,0.0,0.0,0.0,0.01,6592.0,6823.0,23.87,21.96,0.0,3.82,3.73,0.54,4.11,1.86,-0.11,0.0,0.0,3.16,-0.11,5.43,0.0,0.49,0.0,3.65,-11.47,0.0,0.01,-0.4,-0.01,3.74,-0.3,9.42,0.0,0.0,-5.93,-0.06,-1.13,-6.67,-0.11,0.0,3.18,10.35,1354.0,998.0,11562.0,2653.0
```

Comment:
Why is this showing a change? I don't think it should be.

Reply:
@joseph-robertson The problem seems to be caused by not updating [this code](https://github.com/NREL/OpenStudio-HPXML/blob/issue-700/HPXMLtoOpenStudio/resources/hvac.rb#L1061-L1062), which is operating as if the clg_weekday_setpoints/clg_weekend_setpoints variables are still 1D monthly arrays. Can you fix it?